SRC/NBASIC/nbas111e.txt

```diff
@@ -418,26 +415,26 @@ The DOS 2 error codes are the following:
 208: The directory is not empty (when deleting a directory).
 207: Invalid attributes (when changing the attributes of a file/directory).
 206: Invalid "." or ".." operation.
-205: System file exists (when creating a file, any previous existing file is
-     automatically deleted, except the files having the system
+205: System file exists (when creating a file, any previously existing file is
+     automatically deleted, except for the files having 'system'
      attribute set).
-204: System directory exists (same as above).
+204: Directory already exists (same as with 'system' files).
 203: File already exists (when creating a directory).
 202: File is open (when deleting, renaming or moving a file, or when changing
      the file attribute with direct filename specification).
 199: End of file is found (when reading from a file).
 196: Too many opened files (when opening a file).
-195: Invalid file number (greatest than 63).
-194: Invalid file number (never assigned to an opened file).
+195: Invalid file number (greater than 63).
+194: Invalid file number (not assigned to any opened file).
 188: RAM disk already exists (when creating RAM disk).
 187: RAM disk does not exist (when removing RAM disk).
 
 
-5. GRAPHIC COMPRESSION AND DESCOMPRESSION
+5. GRAPHICS COMPRESSION AND DESCOMPRESSION
 
-NestorBASIC includes functions for the compression of graphic data from VRAM 
+NestorBASIC includes functions for compression of the graphic data from VRAM    
 to RAM, and decompression from RAM to VRAM. The compression format is the 
-same as the one used by Sunrise in the logo appearing routine, it is 
+one used by the routine, showing Sunrise logo, it is 

```

Comment:
"the one used by Sunrise to show their logo" is clearer

Reply:
Indeed.